<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgBulkRegister</class>
 <widget class="QDialog" name="CDlgBulkRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>卡批量注册</string>
  </property>
  <property name="statusTip">
   <string>%1卡批量注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(D:/WinParkUI/debug/Image/NewIcon/CommonMiddleBG-normal.jpg);</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url( );
color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>5</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabRecord">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>61</y>
     <width>631</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtCardID">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>620</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="styleSheet">
       <string notr="true">background-image: url(none);</string>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>edtCardID</zorder>
  <zorder>lblTitle</zorder>
  <zorder>lblClose</zorder>
  <zorder>tabRecord</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
